packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -48,7 +48,10 @@ function VisCanvas(props: PropsWithChildren<Props>) {
       }}
     >
       {(!shouldMeasure || canvasSize) && (
-        <div className={styles.canvasWrapper} style={canvasSize}>
+        <div
+          className={styles.canvasWrapper}
+          style={shouldMeasure ? canvasSize : undefined}
```

Comment:
Why is the ternary needed ? Won't `canvasSize` be `undefined` anyway if `shouldMeasure` is `false` ?

Reply:
Unfortunately, no. `useMeasure` doesn't trigger a re-render with `undefined` when it receives `false` after receiving `true`, so `canvasSize` keeps its latest value.

Reply:
Maybe I'll open a PR in react-hookz at some point to fix this.

Reply:
All right then